core/services/fluxmonitor/flux_monitor.go

```diff
@@ -223,8 +216,10 @@ type pollingDeviationCheckerFactory struct {
 }
 
 func (f pollingDeviationCheckerFactory) New(initr models.Initiator, runManager RunManager, orm *orm.ORM) (DeviationChecker, error) {
-	if initr.InitiatorParams.PollingInterval < MinimumPollingInterval {
-		return nil, fmt.Errorf("pollingInterval must be equal or greater than %s", MinimumPollingInterval)
+	minimumPollingInterval := models.Duration(f.store.Config.DefaultHTTPTimeout())
```

Comment:
What do you think about making the timeout an argument?  it would make is requirement explicit.